### ISSUE TITLE
Infinity Ham Alchemy Ability is Confusing

### ISSUE THREAD

**Issue by Zapfreak4** [SSKirillSS/relics#129]
The Alchemy ability on the Infinity Ham says to use a potion on the relic in the inventory but I've tried left and right clicking both on each other and nothings working. Not sure if it's a bug or if I'm doing it wrong, but I wish the description was a bit clearer on how the process worked. I'm on version 1.20.1 of minecraft.

**Comment by catboybinary** [comment]
you might've accidentally tried using an instant or a lingering potion. the ability works only with regular potions, and yeah you should click with a potion on the relic in the inventory, just like a bundle.

**Comment by beckettteich** [comment]
im on 1.21 and this is not working. i have tried strength, jump boost, regen at lvl 1 and 2. left and right click do nothing. its lvl five with 1 into alchemy and 4 into regen. version 7.1.17 ( i think)

**Comment by AlgebraManiacABC** [comment]
Agreed that it should clarify no lingering or instant potions, and should clarify left click. "Use" implies right click.
Left clicking works in Relics 0.8.1 in All The Mods 10 pack (1.21)

**Comment by antisocialian** [comment]
Does this only work with vanilla potions? or any potion that isn't lingering/splash?

**Comment by SSKirillSS** [comment]
Should have been reworked a few updates ago.

**Comment by KeatonSW** [comment]
This is still an issue in the most recent 1.20.1 release, I am using a regular ol' Instant Health II potion both left and right clicking on the infinity ham and still cannot get it to accept the potion for the alchemy ability.
